src/ast/compare.rs

```diff
@@ -448,6 +451,27 @@ pub trait Comparable<'a> {
     where
         T: Into<Expression<'a>>,
         V: Into<Expression<'a>>;
+
+    /// Tests if the value is not between two given values.
```

Comment:
Nitpick: I think this is copy-pasted. Maybe something like `compares two expressions with a custom operator` or something like that.